README.md

```diff
@@ -1,49 +1,82 @@
 # Meetbot 
 
-Meetbot will listen to google meet urls to join to and perform various tasks.
+Meetbot is a Google Meet bot that makes your meetings frictionless. Features include eliminating questions in video calls like:
```

Comment:
I think this section is a bit confusing tbh, or at least it's making it harder to show what features we offer. I like the "eliminating questions" part but I would clearly list the features too.

How about something like this? 

```suggestion
Meetbot is a Google Meet bot that makes your meetings frictionless. We've all been there, we've all heard the same questions being repeated over and over again:

1. _Can you hear me?_
2. _Can you resend those links?_
3. _Do you have the notes from the call?_
4. _Was the meeting recorded?_
5. _Does anyone have the links shared in the meeting?_

Meetbot helps you out by:
1. Validating audio input of attendees minus the awkward silence.
2. Recording chat transcript.
3. Recording voice caption transcript.
4. Auto-recording meetings.
5. Saving it all to Google Docs for easy sharing!
```

Reply:
Much cleaner, I like it too